helm/aladdin-demo/values/values.DEV.yaml

```diff
@@ -0,0 +1,4 @@
+# Values here will overwrite values.yaml defaults. These values are used when deploying to DEV cluster
+
+elasticsearch:
+  create: false
```

Comment:
don't we want to create elasticsearch on DEV?

Reply:
Probably, but this elasticsearch is backed by a statefulset and we have to provision the persistent volumes manually beforehand and then probably specify some nodeAffinity stuff in the values file. I didn't want this example to get out of hand. The explanation for what they have to do for statefulsets is documented.

Reply:
I see. Maybe we should add a comment explaining that then? I can see some team copying this and saying "but the demo project doesn't create ES on dev!!" - or use redis instead since that is simpler?